proxy/contracts/schain/TokenManagers/TokenManagerERC20.sol

```diff
@@ -140,7 +128,7 @@ contract TokenManagerERC20 is TokenManager {
 
         contractOnSchain.burn(amount);
```

Comment:
Maybe we can make _exit function as you added in ERC721/Eth?

Reply:
Changed.